README.md

```diff
@@ -107,23 +107,30 @@ make
 
 GTDynamics now supports a Pybind11-based Python API.
 
-To start, please download and install the [GTwrap repository](https://github.com/borglab/wrap).
+GTWrap comes with the installation with GTSAM, which can also be used for generate the python bindings for GTDynamics.
```

Comment:
This sentence does not make sense from an outsider's perspective. What can GTWrap "also be used" as an alternative to?

Perhaps do something like this?

```suggestion
#### GTWrap Prerequisite

To start, please download and install the [GTwrap repository](https://github.com/borglab/wrap).

GTWrap also comes bundled with GTSAM. Ensure that GTSAM is built with the generated python bindings. To do so, go to the build directory and run `cmake` with the flag `GTSAM_BUILD_PYTHON=ON`. Afterwards, install GTSAM along with its python package.

	```sh
	cmake -DGTSAM_BUILD_PYTHON=ON ..
	make && make install && make python-install
	```

#### GTDynamics Python Wrapper

>>> Add the rest of the instructions here.
```

Reply:
I specifically asked Jerred to *not* use a separate GTWrap implementation. If offered as an alternative, please create a totally different section, and let us not confuse the common use case.